doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1807,6 +1807,9 @@ Response:
 Creates a new subscription.
 The subscription is represented by a JSON object as described at the beginning of this section.
 
+Using `expires` with value equal to empty string (`""`) equals to omit the field, i.e. creating
```

Comment:
On @kzangeli behalf, from the old PR:

equal-equals ... I don't like it very much.

I propose (with a few more 'tweaks'):

    Using `expires` with a value of the empty string (`""`) equals omitting the field, i.e. creating ...